include/velocypack/Buffer.h

```diff
@@ -49,6 +49,8 @@ class Buffer {
     initWithNone();
   }
 
+  Buffer(char *data, uint64_t len) noexcept : _buffer((unsigned char*) data), _capacity(len), _size(len), immutable(true) {}
```

Comment:
why do you need an immutable Builder ? Kind of seems like a contradiction in itself